<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="shah4corp" failures="1" tests="1" name="tutorialsninja.register.TestCaseRegisterFunctionality019" time="12.082" errors="0" timestamp="2025-07-12T12:47:47 IST" skipped="0">
  <testcase classname="tutorialsninja.register.TestCaseRegisterFunctionality019" name="verifyLeadingAndTrailingSpacesWhileRegisteringAccount" time="12.082">
    <failure message="The following asserts failed:
	expected [Arun] but found [        Arun     ],
	expected [Motoori] but found [       Motoori    ],
	expected [1234567890] but found [   1234567890   ]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Arun] but found [        Arun     ],
expected [Motoori] but found [       Motoori    ],
expected [1234567890] but found [   1234567890   ]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at tutorialsninja.register.TestCaseRegisterFunctionality019.verifyLeadingAndTrailingSpacesWhileRegisteringAccount(TestCaseRegisterFunctionality019.java:62)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at org.testng.internal.thread.graph.TestNGFutureTask.run(TestNGFutureTask.java:22)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </failure>
  </testcase> <!-- verifyLeadingAndTrailingSpacesWhileRegisteringAccount -->
</testsuite> <!-- tutorialsninja.register.TestCaseRegisterFunctionality019 -->
